content/help/ops/security/authorization-restrictive/index.md

```diff
@@ -13,6 +13,9 @@ matching requests should flow through. If all requests continue to be denied, yo
 
 1. Make sure that your `ServiceRoleBinding` and referred `ServiceRole` objects are in the same namespace (by checking "metadata"/”namespace” line).
 
+1. Make sure that your service role and service role binding don't use any HTTP only fields for a TCP service,
+otherwise the policy will be ignored as if there is no such policy.
```

Comment:
Active voice, missing nouns, and grammar:
 Make sure that your service role and service role binding policies don't use any HTTP only fields for TCP services. Otherwise, Istio ignores the policies as if they didn't exist.